src/Unosquare.PassCore.Web/Helpers/IPasswordChangeProvider.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Unosquare.PassCore.Web.Helpers
+{
+    using Microsoft.Extensions.Options;
+    using System.DirectoryServices.AccountManagement;
+    using Unosquare.PassCore.Web.Models;
+
+    public interface IPasswordChangeProvider
+    {
+        ApiErrorItem PerformPasswordChange(ChangePasswordModel model, AppSettings options);
```

Comment:
Interface should expose only `ApiErrorItem PerformPasswordChange(ChangePasswordModel model)` method. Remove two below.